src/Tuura/Concept/FSM/Translation.hs

```diff
@@ -301,44 +351,45 @@ expandX xs = case elemIndex triX (srcEncx xs) of
         where
           makeArc s = FsmArcX s (transx xs)
 
--- Replace all TriXs with a 0 and 1 in every source and destination encoding.
+-- | Replace all TriXs with a 0 and 1 in every source and destination encoding.
+-- This creates 2^n states, where n is the number of "don't cares".
 expandAllXs :: [FsmArcX a] -> [FsmArcX a]
 expandAllXs = concatMap expandX
 
--- http://stackoverflow.com/questions/5921573/convert-a-string-representing-a-binary-number-to-a-base-10-string-haskell
--- covert string to int
+-- | http://stackoverflow.com/questions/5921573/convert-a-string-representing-a-binary-number-to-a-base-10-string-haskell
```

Comment:
I would remove the link from the Haddock, but keep it in the comment (it's an implementation detail/reference).